wiki/actuation/Pure-Pursuit-Controller-for-Skid-Steering-Robot.md

```diff
@@ -0,0 +1,69 @@
+---
+# Simple Pure Pursuit based Controller for Skid Steering Robot
+
+---
+This article will cover the steps for implementing a simple Pure Pursuit based controller for a skid steering robot on a flat surface, limiting the degrees of freedom to x, y and heading(yaw).
+The controller will make use of the Pure Pursuit Algorithm to follow a desired trajectory. This is a good method to quickly implement a robust path/trajectory tracking controller for robots travelling at low speeds, where exact trajectory tracking is not required. 
+One can make use of optimal control methods like iLQR for better performance in those cases. 
+The aim of the controller is to determine the desired velocity of the robot given the current location of the robot and the trajectory to be followed.
+The implementation discussed here is based on R. Craig Coulter's  work at the Robotics Institute, Carnegie Mellon University in January, 1992.
```

Comment:
If these are separate paragraphs, Markdown needs a line between them to render correctly.